src/test/stepFunctions/commands/visualizeStateMachine.test.ts

```diff
@@ -24,11 +24,11 @@ const mockGlobalStorage: vscode.Memento = {
 }
 
 const mockUriOne: vscode.Uri = {
-    authority: 'MockAuthorityOne',
+    authority: 'MockAuthorityOne.com',
```

Comment:
```
dig MockAuthorityOne.com

; <<>> DiG 9.10.6 <<>> MockAuthorityOne.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 49090
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;MockAuthorityOne.com.		IN	A

;; AUTHORITY SECTION:
com.			900	IN	SOA	a.gtld-servers.net. nstld.verisign-grs.com. 1598308041 1800 900 604800 86400
```

Reply:
Can we make this amazon.com or something owned by amazon?

Reply:
done